cloud/pkg/devicecontroller/config/config.go

```diff
@@ -48,12 +48,17 @@ func InitConfigure() {
 
 		km, err := config.CONFIG.GetValue("devicecontroller.kube.master").ToString()
 		if err != nil {
-			errs = append(errs, fmt.Errorf("get devicecontroller.kube.master configuration key error %v", err))
+			km = ""
+			klog.Infof("can not get devicecontroller.kube.master key, use default value %v", km)
 		}
 		kc, err := config.CONFIG.GetValue("devicecontroller.kube.kubeconfig").ToString()
 		if err != nil {
 			errs = append(errs, fmt.Errorf("get devicecontroller.kube.kubeconfig configuration key error %v", err))
 		}
+		if km == "" && kc == "" {
```

Comment:
if run cloudcore in container, both km and kc can be empty

Reply:
ok， i only add warnning log